opencti-platform/opencti-graphql/src/modules/report/report.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Placeholder for a report Module Definition
+ */
+
+import type { OverviewLayoutCustomization } from '../entitySetting/entitySetting-types';
+import { registerEntityOverviewLayoutCustomization } from '../../schema/overviewLayoutCustomization-register';
+import { ENTITY_TYPE_CONTAINER_REPORT } from '../../schema/stixDomainObject';
+
+const reportDefaultOverviewLayout: OverviewLayoutCustomization[] = [
+  { key: 'details', width: 6, label: 'Entity details' },
+  { key: 'basicInformation', width: 6, label: 'Basic information' },
+  { key: 'externalReferences', width: 6, label: 'External references' },
+  { key: 'mostRecentHistory', width: 6, label: 'Most recent history' },
+  { key: 'notes', width: 12, label: 'Notes about this entity' },
+];
+
+// Register only the default layout
+registerEntityOverviewLayoutCustomization(
+  ENTITY_TYPE_CONTAINER_REPORT,
+  reportDefaultOverviewLayout,
+);
```

Comment:
For reports, no backend module is coded yet ; still old architecture.

I chose to bootstrap a module file, only declaring the default layout.
We can start from there if/when we move reports to the module architecture.